orttraining/orttraining/test/python/orttraining_test_transformers.py

```diff
@@ -0,0 +1,190 @@
+
+            sequence_labels = None
+            token_labels = None
+            choice_labels = None
+            if self.use_labels:
+                sequence_labels = ids_tensor([self.batch_size], self.type_sequence_label_size).to(self.device)
+                token_labels = ids_tensor([self.batch_size, self.seq_length], self.num_labels).to(self.device)
+                choice_labels = ids_tensor([self.batch_size], self.num_choices).to(self.device)
+
+            config = BertConfig(
+                vocab_size=self.vocab_size,
+                vocab_size_or_config_json_file=self.vocab_size,
+                hidden_size=self.hidden_size,
+                num_hidden_layers=self.num_hidden_layers,
+                num_attention_heads=self.num_attention_heads,
+                intermediate_size=self.intermediate_size,
+                hidden_act=self.hidden_act,
+                hidden_dropout_prob=self.hidden_dropout_prob,
+                attention_probs_dropout_prob=self.attention_probs_dropout_prob,
+                max_position_embeddings=self.max_position_embeddings,
+                type_vocab_size=self.type_vocab_size,
+                is_decoder=False,
+                initializer_range=self.initializer_range)
+
+            return config, input_ids, token_type_ids, input_mask, sequence_labels, token_labels, choice_labels
+
+        def create_and_check_bert_for_pretraining(self, config, input_ids, token_type_ids, input_mask, sequence_labels, token_labels, choice_labels):
+            model = BertForPreTraining(config=config)
+            model.eval()
+            loss, prediction_scores, seq_relationship_score = model(input_ids, attention_mask=input_mask, token_type_ids=token_type_ids,
+                                                                    masked_lm_labels=token_labels, next_sentence_label=sequence_labels)
+            model_desc = ModelDescription([self.input_ids_desc, self.attention_mask_desc, self.token_type_ids_desc,
+                                           self.masked_lm_labels_desc, self.next_sentence_label_desc],
+                                          [self.loss_desc, self.prediction_scores_desc, self.seq_relationship_scores_desc])
+
+            from collections import namedtuple
+            MyArgs = namedtuple("MyArgs", 
+                "local_rank world_size max_steps learning_rate warmup_proportion batch_size seq_len")
+            args = MyArgs(local_rank=0, world_size=1, max_steps=100, learning_rate=0.00001, warmup_proportion=0.01, batch_size=13, seq_len=7)
+
+            def get_lr_this_step(global_step):
+                return get_lr(args, global_step)
+            loss_scaler = LossScaler('loss_scale_input_name', True, up_scale_window=2000)
+
+            option_gradient_accumulation_steps = [8]
+            option_fp16 = [True]
+            option_allreduce_post_accumulation = True
+            option_use_internal_get_lr_this_step = False
+            option_use_internal_loss_scaler = True
```

Comment:
Setting this flag to True, ort_trainer.py:723 is not executed and the logic change is not exercised (` if self.loss_fn_ is None and len(input_desc_with_) > len(input) and len(extra_kargs_keys) == 1 and internal_loss_scale is None:`)

Is that change on ort_trainer.py really necessary or dead code?